test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/QueryExecution/SqlCmdExecutionTest.cs

```diff
@@ -15,31 +15,31 @@ namespace Microsoft.SqlTools.ServiceLayer.IntegrationTests.QueryExecution
 {
     public class SqlCmdExecutionTest
     {
-        [Test]
```

Comment:
please do not comment out code, you can use the `[Ignore("reason goes here")]` attribute instead